mypyc/irbuild/util.py

```diff
@@ -28,7 +28,14 @@
     Var,
 )
 
-DATACLASS_DECORATORS = {"dataclasses.dataclass", "attr.s", "attr.attrs"}
+DATACLASS_DECORATORS = {
+    "dataclasses.dataclass",
+    "attr.s",
+    "attr.attrs",
+    "attrs.define",
```

Comment:
For some reason, the MemberExpr for these have `fullname` set to `attr.define` which isn't included here... do you know why that's the case? If you remove the `slots=False` argument, no error is raised. Are you sure mypyc is recognizing attrs' modern APIs?

```python3
import attrs

@attrs.define(slots=False)
class Person:
    name: str

@attrs.define(slots=False)
class MyException(Exception):
    level: int

    def __str__(self) -> str:
        return f"level {self.level} storm up ahead!"
```

(Also subclassing from Exception seems to be broken, but it's broken on main too.)

Reply:
I can't reproduce it. What version of attrs do you have installed in your environment?

Reply:
(BTW, not sure how to assert a 'compilation error' in those tests.)